docs/docs/domain.mdx

```diff
@@ -384,6 +384,36 @@ class NumberOfPeopleSlot(Slot):
         return r
 ```
 
+A custom slot class is implemented as an independent python module, 
+separate from custom action code. It should be saved in a directory 
+alongside an empty file called "\_\_init\_\_.py" so that it will be recognized as a python module.
+You can then refer to the custom slot class by it's module path.
+
+For example, say the code above is saved in "addons/my_custom_slots.py", a directory relative to your bot project:
+
+```bash
+└── rasa_bot
+    ├── addons
+    │   ├── __init__.py
+    │   └── my_custom_slots.py
+    ├── config.yml
+    ├── credentials.yml
+    ├── data
+    ├── domain.yml
+    ├── endpoints.yml
+```
+
+The custom slot type's module path
+is then `addons.my_custom_slots.NumberOfPeopleSlot`.
+Use the module path to refer to the custom slot type in your domain file:
+
+```yaml-rasa title="domain.yaml"
+slots:
+  people:
+    type: addons.my_custom_slots.NumberOfPeopleSlot
+    influence_conversation: true
+```
+
 Rasa Open Source needs some training stories to learn from how to handle the
```

Comment:
I know you didn't add this, but could you rewrite this please? I don't think this is correct english 😅 . Also, we could explain what we mean by "situations".